src/test/java/ancaMarian/tests/tema8/tests/DisabledDynamicButtonsPageTest.java

```diff
@@ -0,0 +1,28 @@
+package ancaMarian.tests.tema8.tests;
+
+import ancaMarian.tests.tema8.pageObjects.DisabledDynamicButtonsPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class DisabledDynamicButtonsPageTest extends BaseTest {
+
+    @Test(groups = {"mobile", "desktop"})
+    public void dynamicButtonsPageTest() {
+        driver.get("https://testpages.herokuapp.com/styled/dynamic-buttons-disabled.html");
+        DisabledDynamicButtonsPage disabledDynamicButtonsPage = new DisabledDynamicButtonsPage(driver);
+        disabledDynamicButtonsPage.waitForStartButtonToBeClickable();
+        disabledDynamicButtonsPage.clickOnStartButton();
+        disabledDynamicButtonsPage.waitForOneButtonToBeClickable();
+        disabledDynamicButtonsPage.clickOnOneButton();
+        disabledDynamicButtonsPage.waitForWaitMessageToDisappear();
+        disabledDynamicButtonsPage.clickOnTwoButton();
+        disabledDynamicButtonsPage.waitForWaitMessageToDisappear();
+        disabledDynamicButtonsPage.clickOnThreeButton();
+        WebElement confirmationMessage = driver.findElement(By.id("buttonmessage"));
+        Assert.assertEquals(confirmationMessage.getText(), "All Buttons Clicked");
```

Comment:
Please also double check this scenario as it seems that the assertion is failing ->

java.lang.AssertionError: 
Expected :All Buttons Clicked
Actual   :Click Buttons In Order
<Click to see difference>


	at org.testng.Assert.fail(Assert.java:110)
	at org.testng.Assert.failNotEquals(Assert.java:1577)
	at org.testng.Assert.assertEqualsImpl(Assert.java:149)
	at org.testng.Assert.assertEquals(Assert.java:131)
	at org.testng.Assert.assertEquals(Assert.java:655)
	at org.testng.Assert.assertEquals(Assert.java:665)
	at ancaMarian.tests.tema8.tests.DisabledDynamicButtonsPageTest.dynamicButtonsPageTest(DisabledDynamicButtonsPageTest.java:26)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
	at org.testng.SuiteRunner.run(SuiteRunner.java:330)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
	at org.testng.TestNG.runSuites(TestNG.java:1099)
	at org.testng.TestNG.run(TestNG.java:1067)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)